benchmark/C/Savina/piprecision/PiPrecisionBenchmarkGeneratorNoRunner2.lf

```diff
@@ -0,0 +1,267 @@
+   
+    typedef struct WorkMessage {
+      int scale;
+      int term;
+    } WorkMessage;
+    
+    void printBenchmarkInfo(char* benchmarkId) {
```

Comment:
`printBenchmarkInfo()` and `printSystemInfo()` are repeated a lot in various benchmarks. Should we abstract them into a `benchmark_util.h/c`?

Reply:
Good idea! As stated above, these benchmarks are still in a pretty rough shape. The key reason for the PR are other fixes that are in this branch, not so much the benchmarks themselves. We'll improve them later.